tutorials/mongoose/query-string.md

```diff
@@ -0,0 +1,25 @@
+Mongoose's `Model.find()` function is an important method to understand.
+You may call it without any arguments and it will return all the documents in that model.
+You may hone the search by passing in a `filter` which tells mongoose what to look for in the database.
+This `filter` may be an `objectId` or an `object`.
```

Comment:
Filter must be an object.

Reply:
![capture](https://user-images.githubusercontent.com/39510674/135296277-4710390c-09a3-4798-9e1f-d8b7b9126976.png)
You say on the docs site it can be object or objectid